scalableminds/webknossos
# in project create form, add default for isBlacklistedFromReport

### URL of deployed dev instance (used for testing):
- https://fixprojectcreation.webknossos.xyz

### Steps to test:
- in admin→projects→Add Project, create new project, but do not touch the hide-from-report checkbox
- project creation should succeed

### Issues:
- fixes #3335 

------
